Week1/database.js

```diff
@@ -0,0 +1,93 @@
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'world',
+});
+
+function createQuery(query) {
+  connection.query(query, function(error, results, fields) {
+    if (error) {
+      throw error;
+    }
+    console.log(`The Querys is : ${query}
+    `);
+    console.log('The result', results, '\n');
+  });
+}
+
+function insertValues(createdQuery) {
+  for (let i = 0; i < createdQuery.length; i++) {
+    createQuery(createdQuery[i]);
+  }
+}
+
+connection.connect();
+
+const create_country =
+  "CREATE TABLE country (Name VARCHAR(60) NOT NULL ,Continent enum('Asia','Europe','North America','Africa','Australia','Antarctica','South America') NOT NULL ,Region VARCHAR(30),SurfaceArea INT NOT NULL, IndepYear INT,Population DECIMAL(5,2) NOT NULL , LifeExpectancy DECIMAL(5,2),GNP BIGINT,GNPOld INT,LocalName VARCHAR(30),GovernmentForm VARCHAR(50),HeadOfState VARCHAR(50),Capital VARCHAR(50))";
+createQuery(create_country);
+
+const create_city =
+  'CREATE TABLE city (ID INT NOT NULL AUTO_INCREMENT,Name VARCHAR(60) NOT NULL, CountryCode INT NOT NULL, District VARCHAR(60), Population INT NOT NULL , PRIMARY KEY (ID) )';
+createQuery(create_city);
+
+const insert_country_queries = [
+  "INSERT INTO country VALUES ('Netherlands', 'Europe', 'Western Europe', 42508 , 1581, 17.08, 81.51, 901900000, NULL, 'Holland', 'Constitutional Monarchy', 'King Willem-Alexander', 'Amsterdam')",
+  "INSERT INTO country VALUES ('Turkey', 'Asia', 'Europe', 783562 ,1923, 79.81 ,75.76, 2225000000, NULL,'Türkiye','Republic', 'Recep Tayyip Erdogan','Ankara')",
+  "INSERT INTO country VALUES ('Afghanistan', 'Asia', 'Southern and Central Asia', 652237, 1919, 35.53, 63.67, 71050000, NULL, 'Islamic Emirate of Afghanistan', 'Islamic republic','Ashraf Ghani', 'Kabul')",
+  "INSERT INTO country VALUES ('Canada', 'North America', 'North America', 9985000, 1867, 37.06,82.30, 1679000000,NULL, 'Canada', 'Constitutional Monarchy', 'Queen Elizabeth II','Ottowa')",
+  "INSERT INTO country VALUES ('Germany', 'Europe', 'Europe',357386, 1990, 82.79, 80.64, 4280000000 , NULL, 'Deutschland', 'Republic','Frank-Walter Steinmeier', 'Berlin')",
+];
+insertValues(insert_country_queries);
+
+const insert_city_queries = [
+  "INSERT INTO city SET name='Amsterdam', CountryCode=31, District='North Holland', Population= 821752",

```

Comment:
I like that you tried something different, yes it's working and you can reuse this way to run update queries too.
Plus you are using the Auto increment that way!